plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/query/DynamicTableBuilder.java

```diff
@@ -41,7 +41,7 @@
     private static final CalciteSqlCompiler REQUEST_COMPILER = new CalciteSqlCompiler();
     private static final String COLUMN_KEY = "column";
     private static final String WILDCARD = "*";
-    public static final Set<String> DOUBLE_AGGREGATIONS = ImmutableSet.of("distinctcounthll", "avg");
+    public static final Set<String> DOUBLE_AGGREGATIONS = ImmutableSet.of("sum", "distinctcounthll", "avg");
```

Comment:
sum is defined for doubles, but also for bigint, decimal. is it an issue?

Reply:
Pinot client returns sum(<column>) as double regardless of it's type.